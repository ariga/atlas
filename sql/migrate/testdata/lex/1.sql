CREATE TABLE t1(id int);


CREATE TABLE t2(id int);

CREATE TABLE t3(id int);

CREATE TABLE t4(
    id int,
    name varchar(255)
);

CREATE TABLE t4(
    id int,
    `name` varchar(255) DEFAULT ';'
) ENGINE=InnoDB;

CREATE TABLE t5(
    id int
    /* comment */
    -- comment
) ENGINE=InnoDB;