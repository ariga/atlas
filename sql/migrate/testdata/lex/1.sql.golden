CREATE TABLE t1(id int);
-- end --
CREATE TABLE t2(id int);
-- end --
CREATE TABLE t3(id int);
-- end --
CREATE TABLE t4(
    id int,
    name varchar(255)
);
-- end --
CREATE TABLE t4(
    id int,
    `name` varchar(255) DEFAULT ';'
) ENGINE=InnoDB;
-- end --
CREATE TABLE t5(
    id int
    /* comment */
    -- comment
) ENGINE=InnoDB;